pyscal/gasoil.py

```diff
@@ -102,7 +102,7 @@ def __init__(
         self.sgcr = truncate_zeroness(sgcr, name="sgcr")
         self.sgro = truncate_zeroness(sgro, name="sgro")
 
-        if not (np.isclose(self.sgro, 0) or np.isclose(self.sgro, self.sgcr)):
+        if not (np.isclose(self.sgro, 0) or np.isclose(self.sgro - self.sgcr, 0)):
```

Comment:
Out of interest: Why is `np.isclose(self.sgro - self.sgcr, 0)` necessary compared to `np.isclose(self.sgro, self.sgcr)`?